config/default.yml

```diff
@@ -59,3 +50,15 @@ Ezcater/StyleDig:
 Ezcater/RubyTimeout:
   Description: "Disallow use of `Timeout.timeout` because it is unsafe and can cause unexpected behavior."
   Enabled: true
+
+GraphQL/FieldDescription:
+  Enabled: false
+GraphQL/ArgumentDescription:
+  Enabled: false
+GraphQL/ObjectDescription:
+  Include:
+    - "**/types/**/*.rb"
```

Comment:
I left this wide open because I don't think we want to define that the structure is `app/graphql/**`. We're using packwerk in marketplace hatchery so the directory looks more like `app/packages/<packagename>/graphql`.